jinja2/includes/icons/social/discourse.svg

```diff
@@ -0,0 +1,2 @@
+<svg class="icon icon-discourse" xmlns="http://www.w3.org/2000/svg" width="17px" height="19px" viewBox="0 0 11 11"><path d="M5.484.105C2.508.105 0 2.473 0 5.395v5.48l5.484-.004c2.98 0 5.395-2.46 5.395-5.383 0-2.922-2.418-5.383-5.395-5.383zm0 0"/><path d="M5.54 2.152c-1.18 0-2.27.61-2.872 1.602a3.229 3.229 0 0 0-.066 3.242l-.606 1.91 2.172-.484A3.39 3.39 0 0 0 7.855 7.8a3.24 3.24 0 0 0 .79-3.59 3.35 3.35 0 0 0-3.102-2.059zm0 0"/><path d="M8.191 3.438a3.23 3.23 0 0 1-.039 4.023 3.382 3.382 0 0 1-3.984.957l-2.172.488 2.211-.258c1.465.844 3.344.473 4.363-.859a3.236 3.236 0 0 0-.379-4.351zm0 0"/><path d="M7.578 2.832a3.233 3.233 0 0 1 .387 4.008c-.817 1.265-2.41 1.824-3.86 1.36l-2.109.706 2.172-.484c1.547.687 3.371.125 4.242-1.305a3.24 3.24 0 0 0-.832-4.285zm0 0"/><path d="M2.8 7.066a3.243 3.243 0 0 1 1.196-3.96 3.396 3.396 0 0 1 4.2.332c-1.063-1.368-3.012-1.7-4.481-.766-1.465.933-1.953 2.816-1.113 4.324l-.606 1.91zm0 0"/><path d="M2.602 6.996a3.232 3.232 0 0 1 .77-4.059 3.398 3.398 0 0 1 4.206-.105 3.39 3.39 0 0 0-4.527-.293 3.242 3.242 0 0 0-.656 4.406l-.399 1.961zm0 0"/></svg>
```

Comment:
This new SVG is a solid icon again - previously, there was a gap in the middle. Can we return to the version with the gap?

![filled-in-discourse](https://user-images.githubusercontent.com/286017/44367146-9105ed80-a494-11e8-88e1-0b59acf790e6.png)

Reply:
This one should do the trick:

```
<svg aria-hidden="true" data-prefix="fab" data-icon="discourse" class="svg-inline--fa fa-discourse fa-w-14" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="currentColor" d="M225.9 32C103.3 32 0 130.5 0 252.1 0 256 .1 480 .1 480l225.8-.2c122.7 0 222.1-102.3 222.1-223.9C448 134.3 348.6 32 225.9 32zM224 384c-19.4 0-37.9-4.3-54.4-12.1L88.5 392l22.9-75c-9.8-18.1-15.4-38.9-15.4-61 0-70.7 57.3-128 128-128s128 57.3 128 128-57.3 128-128 128z"/></svg>
```

Reply:
Shall I run this through Svgomg before committing? Thanks

Reply:
I already did that, you can use it as is. Thanks!